package.json

```diff
@@ -25,7 +25,7 @@
   "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
     "portscanner": "^2.2.0",
-    "vscode-chrome-debug-core": "^6.7.8",
+    "vscode-chrome-debug-core": "^6.7.12",
     "vscode-debugadapter": "^1.31.0-pre.0",
```

Comment:
Should we update vscode-debugadapter to "1.31.0" to match -core?

Reply:
Yes. Thanks for catching that :)